collect_app/src/main/java/org/odk/collect/android/database/DatabaseFormsRepository.java

```diff
@@ -57,10 +66,11 @@ public Form get(Long id) {
     @Nullable
     @Override
     public Form get(String jrFormId, @Nullable String jrVersion) {
-        if (jrVersion != null) {
-            return queryForForm(JR_FORM_ID + "=? AND " + JR_VERSION + "=?", new String[]{jrFormId, jrVersion});
+        List<Form> all = getAll(jrFormId, jrVersion);
+        if (!all.isEmpty()) {
+            return all.get(0);
```

Comment:
This is a change in behavior because now deleted forms will be excluded. That seems like the desired behavior to me. Also note that if there is a formid/version conflict, we'll just get the first one. That hasn't changed.